doc/user_guide/psy_data.rst

```diff
@@ -230,6 +231,25 @@ An executable example for using the GOcean read-only-verification
 library is included in ``examples/gocean/eg5/readonly``, see
 :ref:`gocean_example_readonly`.
 
+
+Read-Only-Verification Library for Generic Fortran
+++++++++++++++++++++++++++++++++++++++++++++++++++
+
+This library is contained in the ``lib/read_only/generic`` directory and
+it must be compiled before linking any existing Fortran code that uses
+read-only verification.
+
+Compilation of the library is done by invoking ``make`` and setting
+the required variables:
+
+.. code-block:: shell
+
+    make F90=ifort F90FLAGS="--some-flag"
+
+This will create a library called ``lib_read_only.a``.
+An executable example for using the generic read-only-verification
+library is included in ``examples/gocean/eg6/``.
```

Comment:
nemo/eg6 rather than gocean?

Reply:
Indeed.

Reply:
I meant ... you are right, it was indeed wrong.